src/components/TextInput/TextInput.js

```diff
@@ -718,21 +744,14 @@ class TextInput extends React.Component<Props, State> {
                   marginBottom="0"
                 />
               </SearchBarWrapper>
+              {!!displayFiatOptionsFirst && !!fiatOptions.length &&
+                this.renderHorizontalOptions(fiatOptions, fiatOptionsTitle)
+              }
```

Comment:
To avoid DRY maybe you could add `data.length` check within `renderHorizontalOptions` itself and return null if no data, then another suggest would be to call `this.renderHorizontalOptions(fiatOptions, fiatOptionsTitle)` before as `const renderedHorizontalOptions` and in then end it would just be `{!!displayFiatOptionsFirst && renderedHorizontalOptions}` or below – `{!displayFiatOptionsFirst && renderedHorizontalOptions}`. What do you think?